### PR TITLE
Add missing content to addon.xml.in and add translations

### PR DESCRIPTION
**Pull request by rbuehlma**

### CODE REVIEW

**Comment by rbuehlma** [comment]
I removed the license text from all header files as now the license.txt (GPLv2) is included. Is that ok or should we keep the license in the headers?

**Comment by trummerjo** [comment]
Sorry I have a lot of  stuff to do at the moment.
My first guess would be to keep the licence headers in these files because i actually copied them from another kodi plugin (Changed a few things if i remember correctly.)

**Comment by rbuehlma** [comment]
No hurry :) All the removed licenses are GPL2 as well. I'll ask ksooo in your PR he can give a hint.

**Comment by ksooo** [comment]
I'm not a lawer, but imo, if a file was (originally) copied, then the header must not be removed.

**Comment by rbuehlma** [comment]
I re-added the licenses (i.e. removed the commit)... Love changing the history ;)

File: pvr.zattoo/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+msgctxt "#37101"
+msgid "Username"
+msgstr ""
+
+msgctxt "#37102"
+msgid "Password"
+msgstr ""
+
+msgctxt "#37103"
+msgid "Favorite channels only"
```
Comment (ksooo) [inline pvr.zattoo/resources/language/resource.language.en_gb/strings.po]: english gb: "favourite"

File: pvr.zattoo/resources/language/resource.language.en_gb/strings.po [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+msgstr "Load only channels which are in the favorite list"
```
Comment (ksooo) [inline pvr.zattoo/resources/language/resource.language.en_gb/strings.po]: same here: "favourite"

**Comment by rbuehlma** [comment]
Thanks ksooo, I did fix the translations

**Comment by Hedda** [comment]
Will this pull request be merged? 

And will then this repository be submitted/moved upstream to https://github.com/kodi-pvr ?

And binary releases of pvr.zattoo added to https://github.com/xbmc/repo-binary-addons ?

Asking as I did pull request #13

**Comment by Hedda** [comment]
@rbuehlma  Please also see related pull request #14

**Pull request closed without merging**